app/views/users/_blogger.html.erb

```diff
@@ -0,0 +1,12 @@
+
+<div>
+    <div class = "photo">
+        <%= image_tag(user.photo) %>
+    </div>
+    <div class="user">
+        <h2 class="name"><%= user.name %></h2>
```

Comment:
- [ ] The user.name should link to the user#show page. Please 🙏🏽 add the function to the name to be able to see the user information.